xml/System.Net.Http/HttpMethod.xml

```diff
@@ -471,8 +471,8 @@
         <ReturnType>System.Net.Http.HttpMethod</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Represents an HTTP PATCH protocol method.</summary>
+        <value>Returns <see cref="T:System.Net.Http.HttpMethod" />.</value>
```

Comment:
```suggestion
        <value>The HTTP PATCH protocol method.</value>
```

https://github.com/dotnet/dotnet-api-docs/wiki/Property-Value says not to include the type. It will be added by the build process.